src/dagent/docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Start from latest GO base image
+FROM golang:latest
```

Comment:
I think it would be good if it could be fixed in a specific version. In the past, there was a problem that 1.14 version and 1.17 version could not be used together.